swift/ios-shoppe-demo/ViewController.swift

```diff
@@ -8,13 +8,15 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class StoreViewController: UICollectionViewController {
```

Comment:
Hey @HaroldFS it looks like you've got and identical class in 2 files: `ViewController.swift` and `StoreViewController.swift`. Was this intentional?